go/ql/lib/change-notes/2025-01-09-beego-orm-models.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* `database` local source models have been added for the `github.com/beego/beego/client/orm` package.
```

Comment:
Since we are modelling a package with multiple different import paths, it might be better/more accurate to just refer to it by name.
```suggestion
* `database` local source models have been added for the Beego ORM package.
```